### PR TITLE
some updates to text, readme, and .gitignore

### PR DESCRIPTION
**Pull request by gregcaporaso**
Open to feedback on new text, links, etc.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -66,47 +69,49 @@
...
+      </h3>
 
       <div class="row">
         <div class="col-md-4">
           <div class="row bottom-pad">
             <div class="col-md-12">
               <h4>
-                <a href="https://docs.qiime2.org/2022.11/interfaces/q2cli/">q2cli</a>
-                <small>the command line interface</small>
+                <a href="https://docs.qiime2.org/2022.11/interfaces/q2cli/">command line interface:</a>
```
Comment (lizgehret) [inline index.html]:
```suggestion
                <a href="https://docs.qiime2.org/2022.11/interfaces/q2cli/">command line interface</a>
```
I'd remove this colon from the title (or add colons to the other two interface titles) - just for consistency's sake

**Review by lizgehret — CHANGES_REQUESTED** [review]
overall this lgtm - one small comment inline!

File: index.html (hunk 1 above)
Comment (gregcaporaso) [reply index.html]: will do, thanks!

**Comment by lizgehret** [comment]
Hey @gregcaporaso do you want these changes included in 2023.2? If not, I can move this to 2023.4 project board!

**Comment by gregcaporaso** [comment]
@lizgehret, thanks for checking. We can hold off on this - it's still a work-in-progress (the scope is expanding a bit from what I was originally thinking).

**Comment by lizgehret** [comment]
Sounds good! I'll add to future release board until the scope is more clearly defined.

**Comment by gregcaporaso** [comment]
I'm going to revisit this as there may be some useful content in here, and I never got around to the big refactor I was planning (though I do still plan for us to circle back to that in the next few months).

**Comment by ebolyen** [comment]
No longer relevant, we've got bigger changes incoming.

**Pull request closed without merging**